TTS/melo_handler.py

```diff
@@ -10,21 +10,46 @@
 
 console = Console()
 
+WHISPER_LANGUAGE_TO_MELO_LANGUAGE = {
+    "<|en|>": "EN_NEWEST",
+    "<|fr|>": "FR",
+    "<|es|>": "ES",
+    "<|zh|>": "ZH",
+    "<|ja|>": "JP",
+    "<|ko|>": "KR",
+}
+
+WHISPER_LANGUAGE_TO_MELO_SPEAKER = {
+    "<|en|>": "EN-Newest",
+    "<|fr|>": "FR",
+    "<|es|>": "ES",
+    "<|zh|>": "ZH",
+    "<|ja|>": "JP",
+    "<|ko|>": "KR",
+}
+
```

Comment:
```suggestion
WHISPER_LANGUAGE_TO_MELO_LANGUAGE = {
    "en": "EN_NEWEST",
    "fr": "FR",
    "es": "ES",
    "zh": "ZH",
    "ja": "JP",
    "ko": "KR",
}

WHISPER_LANGUAGE_TO_MELO_SPEAKER = {
    "en": "EN-Newest",
    "fr": "FR",
    "es": "ES",
    "zh": "ZH",
    "ja": "JP",
    "ko": "KR",
}

```
Let's rather use ISO language codes.

Reply:
changed